### PR TITLE
Update ftp profile title to proper form

### PR DESCRIPTION
**Pull request by dahaic**

### CODE REVIEW

File: RHEL/6/input/profiles/ftp-server.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <Profile id="ftp-server" extends="server">
-<title override="true">ftp</title>
+<title override="true">FTP Server Profile</title>
 <description override="true">This profile is for FTP servers.</description>
```
Comment (shawndwells) [inline RHEL/6/input/profiles/ftp-server.xml]: Can we use this as an opportunity to call out  vsftpd, vs any/all FTP servers?

e.g. 

title:  ``FTP Server Profile`` --> ``FTP Server Profile (vsftpd) ``
descr: ``This is a profile for FTP servers`` --> ``This is a profile for the vsftpd FTP server``

IMHO this is not worth holding up a merge (happy to merge once jenkins passes), but it'd be great to get this updated while this part of the code is being touched :)

Comment (shawndwells) [inline RHEL/6/input/profiles/ftp-server.xml]: whoops - sorry @mpreisler -- I started reviewing, went to do something else, and didn't see you self-assigned this one before commenting!

Comment (mpreisler) [inline RHEL/6/input/profiles/ftp-server.xml]: No worries, the more the merrier. @dahaic want to make the additional changes as part of this PR or separately?

Comment (dahaic) [inline RHEL/6/input/profiles/ftp-server.xml]: Sure, it makes sense - I am going to update the PR

File: RHEL/6/input/profiles/ftp-server.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <Profile id="ftp-server" extends="server">
-<title override="true">ftp</title>
-<description override="true">This profile is for FTP servers.</description>
+<title override="true">FTP Server Profile (vsftpd)</title>
+<description override="true">This is a profile for the vsftpd FTP server</description>
```
Comment (shawndwells) [inline RHEL/6/input/profiles/ftp-server.xml]: Thanks for doing this! Missing a period, then we can merge this.

Comment (dahaic) [inline RHEL/6/input/profiles/ftp-server.xml]: Meh, I have actually removed it, based on random check of other profile (pci-dss one, I guess). We should probably make some simple pass over all descriptions, to make them consistently useful :). But I don't want to pollute this PR.

Comment (shawndwells) [inline RHEL/6/input/profiles/ftp-server.xml]: Well crap, you're right. Will merge this and I'll open a separate PR for complete sentences in our profile descriptions.

**Pull request merged**